FileAPI/blob/Blob-slice.html

```diff
@@ -35,7 +35,24 @@
     type: "content/type",
     desc: "blob2."
   });
+  
+  test_blob(function() {
+    return blob3 = new Blob().slice(0,0,null);
+  }, {
+    type: "null"
```

Comment:
Is `desc` needed here, or where does the test description come from? If you haven't done so, make sure to set up the test environment and run ./serve to test with the changes in https://codereview.chromium.org/2509833002/ so that they pass for certain.